workspaces/ui-v2/src/pages/diffs/EndpointDocumentationPane.tsx

```diff
@@ -40,14 +40,12 @@ export const EndpointDocumentationPane: FC<
   renderHeader,
   ...props
 }) => {
-  const { endpoints, loading } = useEndpoints();
-  const bodies = useEndpointBody(pathId, method, lastBatchCommit);
-  const thisEndpoint = endpoints.find(
-    (i) => i.pathId === pathId && i.method === method
+  const thisEndpoint = useAppSelector((state) =>
+    state.endpoints.results.data?.find(
+      (endpoint) => endpoint.pathId === pathId && endpoint.method === method
+    )
```

Comment:
In this case (and a few other diff page cases) - the loading state is handled in a parent component. In the future we could have selectors that query the paths, and throw errors if there wasn't a loading state / error state to speed up development and avoid us having to build loading + error states everywhere we want to access `AsyncStatus` fields (with the assumption the loading / error states are handled somewhere above the tree)